.NET/Microsoft.Recognizers.Text.DateTime/English/Extractors/EnglishDateExtractorConfiguration.cs

```diff
@@ -178,8 +138,55 @@ public EnglishDateExtractorConfiguration(DateTimeOptions options = DateTimeOptio
             }
 
             ImplicitDateList = implicitDateList;
+
+            var enableMdy = (Options & DateTimeOptions.EnableMdy) != 0;
```

Comment:
Shouldn't this be based on the YAML config + the new config flag from culture initialization?